web/tests/test_message_students.py

```
@@ -0,0 +1,168 @@
+
+    def test_get_message_form(self):
+        """Test that the message form is displayed with enrolled students."""
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, self.student1.username)
+        self.assertContains(response, self.student1.email)
+        self.assertContains(response, self.student2.username)
+        self.assertContains(response, self.student2.email)
+        self.assertContains(response, "All Enrolled Students")
+
+    def test_send_message_to_all_students(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_send_message_to_all_students`

Add return type annotation: `None`

(ANN201)